container/nginx/confs/proxy.tlsconf

```diff
@@ -37,6 +38,12 @@ server {
 
     resolver    8.8.8.8 1.1.1.1 valid=600s;
 
+    # Add Alt-Svc header to negotiate HTTP/3.
+    add_header Alt-Svc 'h3=":443"; ma=86400';
```

Comment:
One thing to note is that currently http3 has multiple draft versions. Clients currently opt in to which draft range they support. Their is currently a draft range that is highly supported and I believe we have to specify those draft numbers in our `Alt-Svc` header in order to use HTTP3 with a client that only supports a specific draft range. 

[Source 1 (Draft compatibility):](https://hacks.mozilla.org/2021/04/quic-and-http-3-support-now-in-firefox-nightly-and-beta/)

> Both the client and server must support the same QUIC and HTTP/3 draft version to connect with each other. For example, Firefox currently supports drafts 27 to 32 of the specification, so the server must report support of one of these versions (e.g., “h3-32”) in Alt-Svc or HTTPS record for Firefox to try to use QUIC and HTTP/3 with that server. Otherwise, we'd be missing out on a bunch of clients that support http3 but we won't be able to serve anything because they only support a specific draft range that we didn't specify in our `Alt-svc` header.

[Source 2 (Nginx Draft range support): ](https://www.nginx.com/blog/our-roadmap-quic-http-3-support-nginx/)
> We implemented multiple draft versions of HTTP/3, from [draft 23](https://datatracker.ietf.org/doc/draft-ietf-quic-http/23/) through [draft 29](https://datatracker.ietf.org/doc/draft-ietf-quic-http/29/) (the most widely implemented version today), and of course the published standard ([draft 34](https://datatracker.ietf.org/doc/draft-ietf-quic-http/34/)).

I'll note that both my sources were published in 2021, so maybe the draft support is much less now and more people are just using the latest standard. 

[Here] is an example of how I added draft range support in the Nginx-Quic image I created while testing.
`add_header alt-svc 'h3-27=":443"; ma=86400, h3-28=":443"; ma=86400, h3-29=":443"; ma=86400, h3=":443"; ma=86400';`

Reply:
That is a sensible point @AmeanAsad. I was initially using it, but ended up cleaning it up after checking https://quic.nginx.org/readme.html. I'll add your header for compatibility with older clients 👍